src/worker.ts

```diff
@@ -143,6 +143,8 @@ function onMessage (
   (async function () {
     let response : ResponseMessage;
     let transferList : any[] = [];
+    const start = message.histogramEnabled === 1 ? process.hrtime.bigint() : null;
```

Comment:
```suggestion
    const start = message.histogramEnabled === 1 ? performance.now() : null;
```

Using `process.hrtime.bigint()` has a cost. And I doubt that cost worth it for LB decision.

Reply:
Agree, let me also see if we can take advantage and abstain of using `hrtime` in favour of `performance.now` on the places where we query this.